homeassistant/components/bosch_alarm/services.yaml

```diff
@@ -0,0 +1,12 @@
+set_date_time:
+  fields:
+    config_entry_id:
+      required: true
+      selector:
+        config_entry:
+          integration: bosch_alarm
+    datetime:
+      required: false
+      example: "2022/11/01 22:15"
```

Comment:
is this the string value of the datetime selector?

Reply:
oh weird, i wonder where i copied that from since that isnt the text format, ill fix that